src/components/root/routes/Routes.js

```diff
@@ -7,6 +7,8 @@ import LandingPage from '../../LandingPage';
 import Register from '../../auth/Register';
 import SignIn from '../../auth/SignIn';
 import SwagRoutes from './SwagRoutes';
+import ForgotPassword from '../../ForgotPassword/ForgotPassword';
```

Comment:
We should put the forgot password and reset password components in the `auth` folder

Reply:
👍